p2composite.example.site/bintray.ant

```diff
@@ -10,10 +10,19 @@
 	<property name="bintray.releases.path" value="" />
 	<property name="bintray.composite.path" value="" />
 	<property name="bintray.zip.path" value="" />
+	<property name="bintray.publish" value="" />
 	-->
 
-	<property name="bintray.url" value="https://dl.bintray.com/${bintray.owner}/${bintray.repo}" />
-	<property name="bintray.package.version" value="${unqualifiedVersion}.${buildQualifier}" />
+	<property name="bintray.url" value="https://dl.bintray.com/${bintray.owner}/${bintray.repo}/${bintray.package}" />
+	<condition property="bintray.package.version" value="${unqualifiedVersion}" else="${unqualifiedVersion}.${buildQualifier}">
```

Comment:
I believe the proper fix would be to pass both `qualifiedVersion` and `unqualifiedVersion` from Maven.

It makes the change much simpler and it does not require to check if `buildQualifier` is set or not.
See example here: https://github.com/vlsi/mat-calcite-plugin/blob/master/MatCalciteRepository/packaging-p2composite.ant#L32-L33

Reply:
@vlsi thanks for the suggestion! It would be great if you could create a PR

Reply:
Here you go: https://github.com/LorenzoBettini/p2composite-bintray-example/pull/9